saleor/graphql/schema.graphql

```diff
@@ -4401,6 +4482,91 @@ type VoucherUpdate {
   voucher: Voucher
 }
 
+type Warehouse implements Node {
+  id: ID!
+  name: String!
+  companyName: String!
+  shippingZones(before: String, after: String, first: Int, last: Int): ShippingZoneCountableConnection!
+  address: Address!
+  email: String!
+  stockSet(before: String, after: String, first: Int, last: Int): StockCountableConnection!
```

Comment:
Is this field here on purpose? It looks like it was automatically added by Graphene.

Reply:
yeah, it's been added automatically

Reply:
Yes in the `Warehouse`'s metaclass we should add `only_fields` where we'll explicitly define which fields should be converted to GraphQL type from the model.